REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
that's also bad, because there's the critical issue for on ZMQ 0.3.3.
https://github.com/JuliaLang/ZMQ.jl/issues/108#issuecomment-238099002
incidentally causing lot's of deprecated warnings for every input on IJulia.
I prefer to use ZMQ 0.3.4.